src/tests/test_image_modifications.cpp

```diff
@@ -499,10 +499,7 @@ BOOST_AUTO_TEST_CASE(test_blit_modification_decoding_invalid_args)
 	environment_setup env_setup;
 
 	modification_queue queue =
-		modification::decode("~BLIT()"
-				     "~BLIT(wesnoth-icon.png,1,-2)"
-				     "~BLIT(wesnoth-icon.png,-1,2)"
-				     "~BLIT(wesnoth-icon.png,-1,-2)");
+		modification::decode("~BLIT()");
```

Comment:
This is completely wrong. The test is supposed to test what happens if the `BLIT` function is given out-of-range coordinates. With your change the test is no longer testing anything at all.

Instead, you should change the 0 below to 3 and adjust the comment on top of the function.

Reply:
Ah. Sorry, I thought the function was supposed to be testing 'invalid' arguments- since negative arguments are no longer invalid, I removed them from the test (leaving no arguments as the only remaining invalid type). 

Anyway, I'll put the negatives back in. I should probably add in a too many args test while I'm at it, if only to help the function live up to its name.